sections/faqs/dom-two-classes.js

```diff
@@ -0,0 +1,13 @@
+import md from 'components/md'
+
+const TwoDomClasses = () => md`
+  ## Why does a DOM node have two classes?
+
+  Each node actually has two classes connected to it;
+
+  One is static per component meaning that each instance of a component has this class, but it hasn't any style attached to it. However, it can be used to help the developer smoothly tweak all instances of a class at once.
+
+  The other is dynamic, and it's this class which contains all the styling for specific components of a class. This is the class you usually modify.
+
+`
```

Comment:
Let's also take some of @mxstbr's suggestions into account:

> The first class (sc-bfhe32) is static per-component, meaning every instance of a <Button /> will have that class. The second is per-dynamic-stylr-fragmet, so all <Button primary /> will have the same one, all <Button secodary /> will have the same one etc.
Note that the first one doesn't actually have any styles attached to it because it's only used for when you interpolate the component (${Button}) and so that in the DevTools you can quickly tweak all Buttons no matter if they're primary, secondary or whatever else

- It'd be great to show `<Button />` and `<Button secondary />` as examples of how the dynamic class can change
- It'd also be cool to show some examples of how the static and dynamic class can look

Apart from that we can also link to this section: https://www.styled-components.com/docs/tooling#better-debugging

Which helps the user to make the static class human readable for development

Reply:
Based on your comment in the thread about the issue I thought you did not want an example. I agree it would be nice to show examples, but you don't want them to be live, or what did you mean by your comment? :)

Reply:
@nilsgoksor I mean examples as in inline code snippets, not as full fledged live editors or code blocks.

It could be included as:

> Given a styled component, for instance `<Button />`, it would render with the same static class every time. But if the styles are changed using interpolations, with something like `<Button secondary />`, then the dynamic class will be a different one, while the static class would remain the same.

Reply:
@nilsgoksor @philpl I agree inline code blocks are enough. Although nobody could gain anything from a live example :)